src/software/world/game_state.h

```diff
@@ -39,25 +39,13 @@ class GameState
         BALL_PLACEMENT
     };
 
-    State state;
-    RestartReason restart_reason;
-    RefboxGameState game_state;
-    std::optional<Ball> ball_state;
-
-    // True if our team can kick the ball during a restart
-    bool our_restart;
-
-    // The point at which the ball should be placed by robots before a restart. See
-    // Robocup SSL Rules 9.2.
-    std::optional<Point> ball_placement_point;
-
     GameState()
-        : state(HALT),
-          restart_reason(NONE),
-          game_state(RefboxGameState::HALT),
-          ball_state(std::nullopt),
-          our_restart(false),
-          ball_placement_point(std::nullopt)
+        : state_(HALT),
+          restart_reason_(NONE),
+          game_state_(RefboxGameState::HALT),
+          ball_state_(std::nullopt),
+          our_restart_(false),
+          ball_placement_point_(std::nullopt)
```

Comment:
What was the reason for these changes?

Reply:
The name of those member variables has been changed. I guess we have conventions about naming those field variables, but not sure where it is

Reply:
this was just to avoid shadowing variables